packages/cli/src/cmds/preview/preview.ts

```diff
@@ -326,6 +349,10 @@ export default class Preview extends YargsCommand {
       core.localDebug(inputs),
       frontendInstallTask?.wait(frontendInstallStartCb, frontendInstallStopCb),
       backendInstallTask?.wait(backendInstallStartCb, backendInstallStopCb),
+      backendExtensionsInstallTask?.wait(
```

Comment:
Does backend npm install depend on backend extension install?

Reply:
No, `backend npm install` does not depend on `backend extension install`.
If we use `npx func extensions install` command to install functions binding, then `backend extension install` will depend on `backend npm install`. However, we use `dotnet build` to install functions binding.

Reply:
Make sense... just curious why in .vscode/tasks.json it has dependency. That could be another improvement.